lib/wallet/account/hot-wallet-account.ts

```diff
@@ -34,7 +40,191 @@ export class HotWalletAccount implements IWallet {
     private readonly networkId: NetworkType,
   ) {}
 
-  public getPrivateKey() {
+  async performCommitReveal(
+    payload: any,
+    revealPriorityFee: IGeneratorSettingsObject["priorityFee"],
+    extraOutputs: IPaymentOutput[] = [],
```

Comment:
For the consistency, I would use `KAS` in both `priorityFee` and `PaymentOutput`:
https://github.com/forbole/kastle/blob/a2516abac119500a0c22aa62fac0038b8290051f/lib/wallet/interface.ts#L12